app/views/examples/grid/index.njk

```diff
@@ -11,73 +11,97 @@
       </p>
     </div>
-    <div class="govuk-grid__item govuk-grid__item--one-quarter">
+    <div class="govuk-layout-col-one-quarter">
       <h2 class="govuk-heading-m">One quarter</h2>
       <p class="govuk-body">
         This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
       </p>
     </div>
-    <div class="govuk-grid__item govuk-grid__item--one-quarter">
+    <div class="govuk-layout-col-one-quarter">
       <h2 class="govuk-heading-m">One quarter</h2>
       <p class="govuk-body">
         This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
       </p>
     </div>
-    <div class="govuk-grid__item govuk-grid__item--one-quarter">
+    <div class="govuk-layout-col-one-quarter">
       <h2 class="govuk-heading-m">One quarter</h2>
       <p class="govuk-body">
         This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
       </p>
     </div>
-    <div class="govuk-grid__item govuk-grid__item--one-quarter">
+    <div class="govuk-layout-col-one-quarter">
       <h2 class="govuk-heading-m">One quarter</h2>
       <p class="govuk-body">
         This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
       </p>
     </div>
+    <div class="govuk-layout-col-three-quarters">
+      <h2 class="govuk-heading-m">Three-quarters</h2>
+      <p class="govuk-body">
+        This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
+      </p>
+    </div>
+    <div class="govuk-layout-col-one-quarter">
+      <h2 class="govuk-heading-m">One quarter</h2>
+      <p class="govuk-body">
+        This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
+      </p>
+    </div>
+    <div class="govuk-layout-col-one-third on-desktop-one-quarter">
+      <h2 class="govuk-heading-m">100% on mobile, 33% on tablet, 25% on desktop</h2>
+      <p class="govuk-body">
+        This guide shows how to make your service look consistent with the rest of GOV.UK. It includes example code and guidance for layout, typography, colour, images, icons, forms, buttons and data.
+      </p>
+    </div>
+    <div class="govuk-layout-col-two-thirds on-desktop-three-quarters">
```

Comment:
I love the functionality of `on-desktop-three-quarters` but I'm worried it doesn't feel like it's part of the grid syntax.

Did we discuss anything like: 
`govuk-grid-col-two-thirds` `d:govuk-grid-col-three-quarters`

Reply:
on-desktop-three-quarters was an example class name. i could have just used 
`@include govuk-layout-col(three-quarters, $at: desktop, $class: "govuk-grid-col-desktop");`